packages/website/app/root.tsx

```diff
@@ -24,6 +25,10 @@ export const links: LinksFunction = () => {
 			rel: 'stylesheet',
 			href: unoCssUrl,
 		},
+		{
+			rel: 'stylesheet',
+			href: globalCssUrl,
```

Comment:
I still insist this should be component-wise css. It should be included globally.